kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh

```diff
@@ -34,6 +34,7 @@ function usage {
   echo "  -v Validate the existence of persistentVolumeClaim, optional."
   echo "  -k Keep what has been previously from cloned https://github.com/oracle/docker-images.git, optional. "
   echo "     If not specified, this script will always remove existing project directory and clone again."
+  echo "  -s Skip the domain image build optional. "
```

Comment:
fix this sentence

Reply:
I replicate the sentences from other existing parameters  such as 
echo "  -e Also create the resources in the generated YAML files, optional."
echo "  -v Validate the existence of persistentVolumeClaim, optional."

Any Suggestion like this 
   echo "  -s Skip the domain image build  (optional)  "

Reply:
comma or parens are ok - but the way it is now is not correct

Reply:
I modified the sentence as follows with an additional comma 
echo "  -s Skip the domain image build, optional. "